cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2524,6 +2538,13 @@ private boolean isPodUpToDate(StatefulSet sts, Pod pod) {
                     ccDeployment != null ? cruiseControl.generateServiceAccount() : null));
         }
 
+        Future<ReconciliationState> cruiseControlAncillaryCm() {
```

Comment:
What is the ancillary configMap for?

Reply:
Logging configuration. When the cruise-control logging configuration is changed in the Kafka resource, the CO will generate a new CM with these changes and respin the CC pod to pick up the changes